libc/epoll.c

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <openenclave/internal/syscall/sys/epoll.h>
+#include <sys/epoll.h>
+
+OE_STATIC_ASSERT(sizeof(struct oe_epoll_event) == sizeof(struct epoll_event));
+// OE_CHECK_FIELD(struct oe_epoll_event, struct epoll_event, data);
```

Comment:
Maybe uncomment this and see if it fails. This constraint should pass.

Reply:
Needed to readd the OE_CHECK_FEILD macro. Done.